website/server/models/group.js

```diff
@@ -349,7 +349,7 @@ schema.methods.isMember = function isGroupMember (user) {
   }
 };
 
-export function chatDefaults (msg, user) {
+export function chatDefaults (msg, user, participants) {
```

Comment:
This is not something that should be added in chatDefaults. This is behavior particular to starting a quest. IMO, it doesn't make sense to add the logic here.